schema/home/context.js

```diff
@@ -1,5 +1,6 @@
 import { create, assign } from "lodash"
 import gravity from "lib/loaders/gravity"
+import artistLoader from "lib/loaders/per_type"
```

Comment:
Here’s some details about the signature of resolver functions http://dev.apollodata.com/tools/graphql-tools/resolvers.html#Resolver-function-signature. Interesting to note is that supposedly the third parameter is supposed to be used for context such as DataLoader instances, so maybe we’ve been using the wrong parameter (4th) this whole time 🤔

If you want to take a look into that, please go for it.